rust/capture/src/v0_request.rs

```diff
@@ -281,7 +281,7 @@ impl RawRequest {
     }
 
     pub fn events(self, path: &str) -> Result<Vec<RawEvent>, CaptureError> {
```

Comment:
ah and that's what Olly meant when he said it should go in the definition of events... 

(the sound of me slowly learning rust 🤣 )